lib/src/destination.dart

```diff
@@ -88,9 +82,7 @@ class Destination {
   //     them via toMap?
   bool wasDeniedByRobotsTxt = false;
 
-  int _hashCode;
```

Comment:
I assume you know this is a good idea? That, for example, `_hashCode` isn't used, or needed. Just double checking.